### ISSUE TITLE
Detail page: layout of title / author / submitted time / vote / follow /  share

### ISSUE THREAD

**Issue by pdpinch** [mitodl/open-discussions#834]
![detailpage](https://user-images.githubusercontent.com/430126/41659146-77b47f6c-7466-11e8-9f64-843846d5bd57.png)

from https://projects.invisionapp.com/d/main#/console/14224393/303520620/preview

**Comment by aliceriot** [comment]
a few questions on this:

- what should the 'follow' button look like after clicking it? currently we change it to be 'unfollow' and highlight it in yellow (example in the screenshot below)
- what does the 'share' button do?
- the design doesn't currently include a report button, are we moving that somewhere else?
- we currently have a few more buttons that are visible only to moderators:

![moderator_buttons](https://user-images.githubusercontent.com/6207644/41864050-9b421098-7876-11e8-80bf-387e684b975b.png)

basically, the delete and remove buttons. what should we do with those?

**Comment by Ferdi** [comment]
- Follow becomes simply "unfollow", no need for change color for now
- Share: the idea is to get the url of the post. Two examples we looked : https://stackoverflow.com/a/6618176/4992607  , https://www.quora.com/What-do-you-think-of-the-MIT-Sloan-Artificial-Intelligence-online-short-course . I'm asking for some designs though
- remove , report ,etc  go under a vertical dotted menu on the far right.  I'll ask him to add in the design

**Comment by aliceriot** [comment]
one more question! is there a design for the 'edit' link?

**Comment by pdpinch** [comment]
@aliceriot did you get the design for the edit link?

**Comment by aliceriot** [comment]
yep
